core/src/test/java/nl/nn/adapterframework/util/FileHandlerTestBase.java

```diff
@@ -29,353 +31,357 @@
 
 public abstract class FileHandlerTestBase {
 
-	public static String BASEDIR="/FileHandler/";
+	public static String BASEDIR = "/FileHandler/";
+	private Logger log = LogUtil.getLogger(this);
+
+	public @Rule TemporaryFolder tempFolder = new TemporaryFolder();
 
-	public String TEST_TEMP_DIR="target";
-	
 	private IFileHandler handler;
-	private PipeLineSession session=new PipeLineSession();
+	private PipeLineSession session = new PipeLineSession();
 
+	protected abstract IFileHandler createFileHandler() throws IllegalAccessException, InstantiationException;
 
-    protected abstract IFileHandler createFileHandler() throws IllegalAccessException, InstantiationException;
-    
-	public String CHARSET_UTF8="UTF-8";
-	public String charset=CHARSET_UTF8;
+	public String CHARSET_UTF8 = "UTF-8";
```

Comment:
Moet dit geen static zijn? Of is deze wel nodig, want overal wordt de charset die hieronder staat gebruikt.